components/operator/Dockerfile

```diff
@@ -31,6 +31,7 @@ WORKDIR /workdir
 ARG PURPOSE
 ARG IMG_DIRECTORY
 ARG IMG_VERSION
+ARG SERVERLESS_CONFIG_VERSION
```

Comment:
IMO this argument should have a default value. It would be better if something call this Dockerfile but we didn't localize such call.

Reply:
you are right @anoipm also it should have another name because the `SERVERLESS_CONFIG_VERSION` suggests that it's about the config version but it keeps the whole path to the chart dir

Reply:
we decided to copy both serverless configurations and remove this argument.